f2/src/forms/ImpactStatementInfoForm.js

```diff
@@ -53,76 +53,90 @@ export const ImpactStatementInfoForm = props => {
     'impactPage.affected7',
   ]
   return (
-    <ApolloConsumer>
-      {client => (
-        <Form
-          initialValues={getImpact(client)}
-          onSubmit={data => props.onSubmit(client, data)}
-          validate={validate}
-          render={({ handleSubmit, values }) => (
-            <Stack
-              as="form"
-              onSubmit={handleSubmit}
-              shouldWrapChildren
-              spacing={6}
-            >
-              <Control as="fieldset" name="howWereYouAffected">
-                <FormLabel as="legend" htmlFor="howWereYouAffected" mb={2}>
-                  <Text fontWeight="bold">
-                    <Trans id="impactPage.detail" />
-                  </Text>
-                </FormLabel>
-                <Stack spacing={4} shouldWrapChildren>
-                  {howWereYouAffected.map(key => {
-                    return (
-                      <CheckboxArrayControl
-                        key={key}
-                        name="howWereYouAffected"
-                        value={key}
-                        isChecked={getImpact(
-                          client,
-                        ).howWereYouAffected.includes(key)}
-                      >
-                        {i18n._(key)}
-                      </CheckboxArrayControl>
-                    )
-                  })}
-                </Stack>
-              </Control>
+    <React.Fragment>
+      {false ? ( // mark ids for lingui
+        <div>
+          <Trans id="impactPage.affected1" />
+          <Trans id="impactPage.affected2" />
+          <Trans id="impactPage.affected3" />
+          <Trans id="impactPage.affected4" />
+          <Trans id="impactPage.affected5" />
+          <Trans id="impactPage.affected6" />
+          <Trans id="impactPage.affected7" />
+        </div>
+      ) : null}
```

Comment:
the changes to this file are basically just the added lines ☝️  that mark the ids for lingui, but that caused an indentation change for all the other lines 😞